app/handlers.py

```diff
@@ -11,8 +11,10 @@ def create_access_token(account_id, payload):
     linked_account_id = payload.get("linked_account_id")
 
     # change this once only 1 account is linked
-    verification_status = metadata["account"].get(
-        "verification_status", "instantly_verified"
+    verification_status = (
+        "instantly_verified"
+        if not metadata["account"].get("verification_status")
+        else metadata["account"].get("verification_status")
```

Comment:
isn't this essentially the same as
`metadata["account"].get(
         "verification_status", "instantly_verified")`

Reply:
for instant auth, the "verification_status" key is still present, but set to none. so when we get it, we are getting the value of none instead of our intended default.

Reply:
that's how `.get` works if `metadata["account"].get( "verification_status")` is `None` then it's going to default to `"instantly_verified"`

Reply:
never mind. It seems like we'll always have `verification_status` but it'll be set to `None`.

Reply:
yup, that's what I noticed for instant auth